front/cypress/e2e/native_survey_page_action.cy.ts

```diff
@@ -81,19 +81,19 @@ describe('Native survey project page actions', () => {
   it('tests actions when continuous survey project is active and accepting submissions', () => {
```

Comment:
This is failing in master. I'm glad the fix is in here. I'll leave it as is there since the fix will come with your changes.

https://github.com/CitizenLabDotCo/citizenlab/blob/master/front/cypress/e2e/native_survey_page_action.cy.ts#L81